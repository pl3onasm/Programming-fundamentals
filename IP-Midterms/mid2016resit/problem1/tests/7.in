45875 12210
