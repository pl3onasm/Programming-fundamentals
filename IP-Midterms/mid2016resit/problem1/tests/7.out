45875-12210=99999-((99999-45875)+12210)=99999-(54124+12210)=99999-66334=33665
